packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -513,6 +514,9 @@ export default class NewVm extends BaseComponent {
       pv_args: state.pv_args,
       autoPoweron: state.autoPoweron,
       bootAfterCreate: state.bootAfterCreate,
+      copyHostBiosStrings:
+        state.hvmBootFirmware !== 'uefi' &&
+        (this._templateHasBiosStrings() || state.copyHostBiosStrings),
```

Comment:
`state.hvmBootFirmware !== 'uefi' && !this._templateHasBiosStrings() && state.copyHostBiosStrings`